packages/editor-ui/src/composables/useWorkflowHelpers.test.ts

```diff
@@ -96,49 +77,47 @@ describe('useWorkflowHelpers', () => {
 		});
 
 		it('should respect `resetWebhookUrls: false` when duplicating workflows', async () => {
-			if (!duplicateTestWorkflow.nodes) {
+			const workflow = getDuplicateTestWorkflow();
+			if (!workflow.nodes) {
 				throw new Error('Missing nodes in test workflow');
 			}
 			const { saveAsNewWorkflow } = useWorkflowHelpers({ router });
-			const webHookIdsPreSave = duplicateTestWorkflow.nodes.map((node) => node.webhookId);
-			const pathsPreSave = duplicateTestWorkflow.nodes.map((node) => node.parameters.path);
+			const webHookIdsPreSave = workflow.nodes.map((node) => node.webhookId);
+			const pathsPreSave = workflow.nodes.map((node) => node.parameters.path);
 
 			await saveAsNewWorkflow({
-				name: duplicateTestWorkflow.name,
+				name: workflow.name,
 				resetWebhookUrls: false,
-				data: duplicateTestWorkflow,
+				data: workflow,
 			});
 
-			const webHookIdsPostSave = duplicateTestWorkflow.nodes.map((node) => node.webhookId);
-			const pathsPostSave = duplicateTestWorkflow.nodes.map((node) => node.parameters.path);
-			const webhookSuffixPostSave = extractWebhookSuffixes(duplicateTestWorkflow.nodes);
+			const webHookIdsPostSave = workflow.nodes.map((node) => node.webhookId);
+			const pathsPostSave = workflow.nodes.map((node) => node.parameters.path);
 			// Expect webhookIds, paths and suffix to be the same as in the original workflow
 			expect(webHookIdsPreSave).toEqual(webHookIdsPostSave);
 			expect(pathsPreSave).toEqual(pathsPostSave);
-			expect(webhookSuffixPostSave).toEqual([TEST_WEBHOOK_SUFFIX]);
 		});
 
 		it('should respect `resetWebhookUrls: true` when duplicating workflows', async () => {
-			if (!duplicateTestWorkflow.nodes) {
+			const workflow = getDuplicateTestWorkflow();
+			if (!workflow.nodes) {
 				throw new Error('Missing nodes in test workflow');
 			}
 			const { saveAsNewWorkflow } = useWorkflowHelpers({ router });
-			const webHookIdsPreSave = duplicateTestWorkflow.nodes.map((node) => node.webhookId);
-			const pathsPreSave = duplicateTestWorkflow.nodes.map((node) => node.parameters.path);
+			const webHookIdsPreSave = workflow.nodes.map((node) => node.webhookId);
+			const pathsPreSave = workflow.nodes.map((node) => node.parameters.path);
 
 			await saveAsNewWorkflow({
-				name: duplicateTestWorkflow.name,
+				name: workflow.name,
 				resetWebhookUrls: true,
-				data: duplicateTestWorkflow,
+				data: workflow,
 			});
 
-			const webHookIdsPostSave = duplicateTestWorkflow.nodes.map((node) => node.webhookId);
-			const pathsPostSave = duplicateTestWorkflow.nodes.map((node) => node.parameters.path);
-			const webhookSuffixPostSave = extractWebhookSuffixes(duplicateTestWorkflow.nodes);
+			const webHookIdsPostSave = workflow.nodes.map((node) => node.webhookId);
+			const pathsPostSave = workflow.nodes.map((node) => node.parameters.path);
 			// Now, expect webhookIds, paths and suffix to be different
```

Comment:
Nit: The comment is no longer up to date since you removed the suffix check

Reply:
Ah yeah, will update